source/stylesheets/main.scss

```diff
@@ -31,17 +30,120 @@
+	src: url("../fonts/35C30F_C_0.eot");
+	src: url("../fonts/35C30F_C_0.eot?#iefix") format("embedded-opentype"),
+		url("../fonts/35C30F_C_0.woff2") format("woff2"), url("../fonts/35C30F_C_0.woff") format("woff"),
+		url("../fonts/35C30F_C_0.ttf") format("truetype"), url("../fonts/35C30F_C_0.svg#wf") format("svg");
+	font-weight: 200;
+}
+
+@font-face {
+	font-family: "TTNorms";
+	src: url("../fonts/35C30F_F_0.eot");
+	src: url("../fonts/35C30F_F_0.eot?#iefix") format("embedded-opentype"),
+		url("../fonts/35C30F_F_0.woff2") format("woff2"), url("../fonts/35C30F_F_0.woff") format("woff"),
+		url("../fonts/35C30F_F_0.ttf") format("truetype"), url("../fonts/35C30F_F_0.svg#wf") format("svg");
+	font-weight: 400;
+}
+
+@font-face {
+	font-family: "TTNorms";
+	src: url("../fonts/35C30F_D_0.eot");
+	src: url("../fonts/35C30F_D_0.eot?#iefix") format("embedded-opentype"),
+		url("../fonts/35C30F_D_0.woff2") format("woff2"), url("../fonts/35C30F_D_0.woff") format("woff"),
+		url("../fonts/35C30F_D_0.ttf") format("truetype"), url("../fonts/35C30F_D_0.svg#wf") format("svg");
+	font-weight: 500;
+}
+
+@font-face {
+	font-family: "TTNorms";
+	src: url("../fonts/35C30F_1_0.eot");
+	src: url("../fonts/35C30F_1_0.eot?#iefix") format("embedded-opentype"),
+		url("../fonts/35C30F_1_0.woff2") format("woff2"), url("../fonts/35C30F_1_0.woff") format("woff"),
+		url("../fonts/35C30F_1_0.ttf") format("truetype"), url("../fonts/35C30F_1_0.svg#wf") format("svg");
+	font-weight: 700;
+}
+
+@font-face {
+	font-family: "TTNorms";
+	src: url("../fonts/35C30F_3_0.eot");
+	src: url("../fonts/35C30F_3_0.eot?#iefix") format("embedded-opentype"),
+		url("../fonts/35C30F_3_0.woff2") format("woff2"), url("../fonts/35C30F_3_0.woff") format("woff"),
+		url("../fonts/35C30F_3_0.ttf") format("truetype"), url("../fonts/35C30F_3_0.svg#wf") format("svg");
+	font-weight: 900;
+}
+
+@font-face {
+	font-family: "SourceCodePro";
+	font-style: normal;
+	font-weight: 200;
+	src: url("../fonts/SourceCodePro-ExtraLight.ttf") format("truetype");
+}
+@font-face {
+	font-family: "SourceCodePro";
+	font-style: normal;
+	font-weight: 300;
+	src: url("../fonts/SourceCodePro-Light.ttf") format("truetype");
+}
+@font-face {
+	font-family: "SourceCodePro";
+	font-style: normal;
+	font-weight: 400;
+	src: url("../fonts/SourceCodePro-Regular.ttf") format("truetype");
+}
+@font-face {
+	font-family: "SourceCodePro";
+	font-style: normal;
+	font-weight: 500;
+	src: url("../fonts/SourceCodePro-Medium.ttf") format("truetype");
+}
+@font-face {
+	font-family: "SourceCodePro";
+	font-style: normal;
+	font-weight: 600;
+	src: url("../fonts/SourceCodePro-Semibold.ttf") format("truetype");
+}
+@font-face {
+	font-family: "SourceCodePro";
+	font-style: normal;
+	font-weight: 700;
+	src: url("../fonts/SourceCodePro-Bold.ttf") format("truetype");
+}
+@font-face {
+	font-family: "SourceCodePro";
+	font-style: normal;
+	font-weight: 900;
+	src: url("../fonts/SourceCodePro-Black.ttf") format("truetype");
+}
```

Comment:
moved font faces here to avoid webpack config magic (when importing these relative urls can not be resolved -> need to sit on the root)